.github/workflows/check_docstrings.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow will install Python dependencies and check docstrings with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Check docstrings
```

Comment:
we could consider integrating this into the tests job, but I think for now leave as is :-)